### ISSUE TITLE
Links of bridged messages from other bots render incorrectly in matrix

### ISSUE THREAD

**Issue by Zwelf** [mautrix/discord#115]
markdown link from bots look wrong on matrix side.

![discord](https://github.com/mautrix/discord/assets/9964758/1a67c296-1583-491e-82ab-5b0e0d621946)
![matrix](https://github.com/mautrix/discord/assets/9964758/148817ca-b9bc-44c1-bc3c-ff0260152f9a)

```json
{
  "m.new_content": {
      "body": "New map [Vigorous](https://ddnet.org/maps/?map=Vigorous) by [Wartoz](https://ddnet.org/mappers/Wartoz/) & [shengyan](https://ddnet.org/mappers/shengyan/) released on [Moderate](https://ddnet.org/ranks/moderate/).",
      "format": "org.matrix.custom.html",
      "formatted_body": "New map [Vigorous](<a href=\"https://ddnet.org/maps/?map=Vigorous\">https://ddnet.org/maps/?map=Vigorous</a>) by [Wartoz](<a href=\"https://ddnet.org/mappers/Wartoz/\">https://ddnet.org/mappers/Wartoz/</a>) &amp; [shengyan](<a href=\"https://ddnet.org/mappers/shengyan/\">https://ddnet.org/mappers/shengyan/</a>) released on [Moderate](<a href=\"https://ddnet.org/ranks/moderate/\">https://ddnet.org/ranks/moderate/</a>).",
      "msgtype": "m.text"
      "..."
    },
  },
```

**Comment by Zwelf** [comment]
So probably fixed in 345391f8b1f28e6fc69c4b414805f8146684fa6a, because I'm running mautrix-discord 0.6.1+dev.fb6d89a8 (Aug 16 2023, 22:01:30 with go1.21.0) (fb6d89a8). Thanks. I'll update my bridge.

**Comment by Zwelf** [comment]
I can confirm, that with the new version this got fixed.
